app/models/vendor.rb

```diff
@@ -1,2 +1,5 @@
 class Vendor < ApplicationRecord
+  validates_presence_of :slug
```

Comment:
Do we want to validate uniqueness of the slug?

Reply:
I left it out so we could discuss.  Thank you for the reminder.  It seems to me that we have to, but I wanted to run it by the group.